docs/RELEASE-NOTES.rst

```diff
@@ -8,6 +8,10 @@ Added Functionality
 ```````````````````
 * Support for BIG-IP partitions with non-zero default route domains.
 
+Bug Fixes
+`````````
+* Adds correct Marathon healthCheck path to BIG-IP Health Monitor Send String  `[#256] <https://github.com/F5Networks/marathon-bigip-ctlr/issues/256>`_
```

Comment:
Per offline chat, please add other bugs that have been fixed either in this codebase or in CCCL to this list.

Reply:
Not all things changed/fixed in CCCL are utilized by this CC (like vlans). Ken already added a line for non-zero default route domain support. The only other visible thing would be that the User-Agent header for ICR requests now contains the version and build info.

Reply:
But the User-Agent stuff only applies to master, not 1.1-stable.

Reply:
In that case, perhaps add the release-note for the User-agent as a separate commit (either here or in a separate PR), and don't cherry-pick that to 1.1-stable. Looks good to me otherwise. Thanks.

Reply:
I pulled the version/build related stuff out of this and will add that as a separate PR to master.